pegasus-all/build.gradle

```diff
@@ -0,0 +1,12 @@
+/*
+    This is a meta-project that programmatically depends on all other consumer-facing modules
```

Comment:
Should we add any documentations to other places as well?

Reply:
I'm not sure. We could add documentation to the website, but there isn't currently documentation on which artifacts do what. We could potentially add documentation to the description of each module's POM artifact, but that's a change for another day.